GoogleMeetup/src/main/webapp/style.css

```diff
@@ -1,9 +1,353 @@
+#navbar.active li a { animation: topItemAnimation 0.5s ease 0.5s backwards;}
+#navbar.active li:nth-of-type(1) a { animation: topItemAnimation 0.5s ease 0.08333333333333333s backwards;}
+#navbar.active li:nth-of-type(2) a { animation: topItemAnimation 0.5s ease 0.16666666666666666s backwards;}
+#navbar.active li:nth-of-type(3) a { animation: topItemAnimation 0.5s ease 0.25s backwards;}
+#navbar.active li:nth-of-type(4) a { animation: topItemAnimation 0.5s ease 0.3333333333333333s backwards;}
+#navbar.active li:nth-of-type(5) a { animation: topItemAnimation 0.5s ease 0.4166666666666667s backwards;}@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) { #navbar.active li a{animation:none!important;}}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+@import url("https://fonts.googleapis.com/css?family=Roboto");
+
+#container {
+    font-family: "Roboto", sans-serif;
+    position: relative;
+    margin-left: 25px;
+    width: 275px;
+    height: 90px;
+}
+
+#container p {
+    position: absolute;
+    margin: 0;
+    bottom: 0px;
+    right: 1px;
+    font-size: 30px;
+    color: #444;
+}
+
+#logo {
+    position: relative;
+    width: 75px;
+    height: 75px;
+    border-radius: 50%;
+    margin-top: 10px;
+    overflow: hidden;
+    border: 7px solid #f1f1f1;
+    box-shadow: 0 0px 2px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+}
+
+#logo::after {
+    position: absolute;
+    content: "";
+    display: block;
+    width: 60%;
+    height: 60%;
+    border-radius: 50%;
+    background: #f1f1f1;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    margin: auto;
+    z-index: 4;
+}
+
+#logo::before {
+    position: absolute;
+    content: "";
+    display: block;
+    width: 35%;
+    height: 32%;
+    background: #f1f1f1;
+    right: 0;
+    z-index: 10;
+    bottom: 50%;
+    transform: rotateZ(45deg);
+    -webkit-transform: rotateZ(45deg);
+}
+
+#logo .g-line {
+    width: 50%;
+    height: 20%;
+    background: #0091ea;
+    position: absolute;
+    right: 0;
+    margin: auto;
+    border-bottom-right-radius: 4px 20px;
+    top: 0;
+    bottom: 0;
+    z-index: 15;
+}
+
+#logo .yellow {
+    width: 40%;
+    height: 40%;
+    position: absolute;
+    left: -15%;
+    bottom: 32%;
+    margin: auto;
+    background: #ffc107;
+    transform: rotateZ(-48deg);
+    -webkit-transform: rotateZ(-48deg);
+    z-index: 3;
+}
+
+#logo .green {
+    position: absolute;
+    width: 100%;
+    height: 50%;
+    bottom: 0;
+    border-radius: 0 0 100% 100%;
+    background: #4caf50;
+    z-index: 2;
+}
+
+#logo .blue {
+    position: absolute;
+    width: 0px;
+    height: 0px;
+    right: 0;
+    z-index: 30;
+    top: 50%;
+    z-index: 2;
+    width: 35%;
+    height: 32%;
+    background: #0091ea;
+    right: 0;
+    z-index: 2;
+    transform: rotateZ(45deg);
+    -webkit-transform: rotateZ(45deg);
+}
+
+#logo .red {
+    width: 81%;
+    height: 50%;
+    position: absolute;
+    top: 0px;
+    background: #f44336;
+}
+
+
```

Comment:
Do we need so many blank lines?

Reply:
Yeah not really, I was just keeping things separate while I was working on the different parts. I'll remove them.